K8s/kafka-deployment.yaml

```diff
@@ -18,31 +18,31 @@ kind: Deployment
 metadata:
   namespace: soda-multi-cloud
   labels:
-    io.soda.service: kafka-svc
-  name: kafka-svc
+    io.soda.service: kafka
+  name: kafka
```

Comment:
IMO, if deployment labels are changes, service labels need to get changed aswell

Reply:
service is already having the right name !